comp/api/authtoken/noneimpl/authtoken.go

```diff
@@ -0,0 +1,46 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package noneimpl implements a noop version of the auth_token component
+package noneimpl
+
+import (
+	"crypto/tls"
+
+	"github.com/DataDog/datadog-agent/comp/api/authtoken"
+)
+
+// // Module defines the fx options for this component.
+// func Module() fxutil.Module {
+// 	return fxutil.Component(
+// 		fx.Provide(newAuthToken),
+// 		fxutil.ProvideOptional[authtoken.Component](),
+// 	)
+// }
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Is that needed ?

Reply:
No need, I just forgot to remove it.